easybuild/easyblocks/n/netcdf.py

```diff
@@ -71,6 +71,27 @@ def configure_step(self):
 
             CMakeMake.configure_step(self)
 
+        if LooseVersion(self.version) >= LooseVersion("4.3.3"):
+            set.cfg.update('configopts', '-DCMAKE_BUILD_TYPE=RELEASE -DCMAKE_C_FLAGS_RELEASE="-DNDEBUG " ')
+            hdf5 = get_software_root('HDF5')
+            hdf5_libdir = get_software_libdir('HDF5')
+            szip_libdir = get_software_libdir('Szip')
+            if hdf5_libdir and szip_libdir:
```

Comment:
why the `and`, rather than having `if hdf5_libdir` and `if szip_libdir` separately?